controllers/volsync/vshandler.go

```diff
@@ -125,7 +121,7 @@ func (v *VSHandler) ReconcileRD(
 		return nil, err
 	}
 
-	copyMethod, dstPVC, err := v.SelectDestCopyMethod(rdSpec, l)
+	copyMethod, dstPVC, err := v.SelectDestCopyMethod(rdSpec)
```

Comment:
Minor comment for readability here - feel free to ignore or fix later on as part of different cleanup:

I think this isn't new to this PR, but perhaps is a remanent of LocalDirect - it seems SelectDestCopyMethod will always return Snapshot, even with the Ramen "CopyMethod" is set to Direct (this is because "Direct" for Ramen means we'll pre-create the destpvc/reuse app pvc and then tell volsync to use it - and we want a snapshot at the end of every replication cycle).

So - nothing wrong with the above - but I think it's just confusing to read atm.  I.e. SelectDestCopyMethod is always going to return Snapshot - So is there any point in returning this copyMethod variable?  It would make sense if v.createOrUpdateRD() was used in the localRD too, but I don't think it is?  I..e createOrUpdateRD will always use snapshot, so this extra copyMethod variable just makes things confusing, especially when it will actually be `Snapshot` when our ramen copy method is `Direct`.

For example these will be logged (in SelectDestCopyMethod):

```
v.log.Info(fmt.Sprintf("Using copyMethod of Direct with App PVC %s", rdSpec.ProtectedPVC.Name))
```

But then you'll also see this logged (in ReconcileRD):

```
l.V(1).Info(fmt.Sprintf("ReplicationDestination Reconcile Complete rd=%s, Copy method: %s",
		rd.Name, v.destinationCopyMethod))
```

^ and this one will say "Copy method: Snapshot"


  Might be nice to rename SelectDestCopyMethod to something like precreateDestPVC(), something along those lines.